tartufo/scanner.py

```diff
@@ -462,21 +462,32 @@ def chunks(self) -> Generator[types.Chunk, None, None]:
         already_searched: Set[bytes] = set()
 
         try:
```

Comment:
I think it makes sense to move this try block to only encapsulate the new `else` clause below, since it's meant to capture errors interacting with remotes (raising `types.GitRemoteException`). Do we know if just checking `self._repo.branches` ends up calling the `git` command? If not, I don't think this particular `try`/`except` is valuable to wrap this code. If so, we probably just need to do eventually raise a slightly different exception for this new behavior.

Hopefully that rambling made some sense. 😄